src/views/Layout.jsx

```diff
@@ -26,7 +26,8 @@ export function Layout() {
 				<div className="Layout text-black dark:text-white bg-white dark:bg-black">
 					<NavBar darkMode={darkMode} toggleDarkMode={toggleDarkMode} />
 
-					<main className="mx-auto px-8 md:px-8 lg:px-20 w-full max-w-screen-sm min-h-screen flex flex-col">
+					<main className="mx-auto px-8 md:px-8 lg:px-20 w-full max-w-screen-sm min-h-screen flex flex-col rounded-xl pt-20">
```

Comment:
Adding bottom padding here would be good so main isn't covered by the nav bar when you scroll to the bottom